tests/e2e/Services/Projects/ProjectsConsoleClientTest.php

```diff
@@ -2156,6 +2156,53 @@ public function testGetProjectPlatform($data): array
         $this->assertEquals('com.example.macos', $response['body']['key']);
         $this->assertEquals('', $response['body']['store']);
         $this->assertEquals('', $response['body']['hostname']);
+        
+        public function testCreateProjectDomain($data): array 
+        { 
+         $id = $data['projectId'] ?? ''; 
```

Comment:
Looks like there are style problems indicated by the lint errors. Would you please fix them so the CI job passes?